extensions/Bookalope/js/uses/Spectrum.js

```diff
@@ -0,0 +1,303 @@
+/*jslint browser: true, devel: true */
+/*global window, document, navigator, atob */
+
+/**
+ * Alright this is a funky Adobe Spectrum thing. Looking at the documentation for Picker
+ * elements:
+ *
+ *   https://opensource.adobe.com/spectrum-css/picker.html
+ *   https://spectrum.adobe.com/page/picker/
+ *
```

Comment:
Perhaps we should add this one link to justify the entirety of the JavaScript file 🤓
```
https://github.com/adobe/spectrum-css#where-is-the-javascript
```